tests/tests_ntp.yml

```diff
@@ -1,4 +1,4 @@
-
+---
```

Comment:
This should not be needed.
```yaml
  document-start: disable
```
is specified in the yamllint configuration.

Reply:
The error was this:
```
./tests/tests_ntp.yml
  1:1       error    too many blank lines (1 > 0)  (empty-lines)
```
That is, the file begins with a blank line.  I guess `document-start: disable` is meant to ignore when the file starts with the actual yaml instead of the `---`?  Because if the file looks like this:
```yaml
- name: Configure time synchronization with NTP servers
  hosts: all
...
```
(note: no blank line at the beginning of the file), then `yamllint` reports no errors - the `document-start: disable` directive ignores that the file doesn't start with `---`.

Reply:
It took only a few minutes to add `---` to the beginning of the files that reported this error, so I just went ahead and did it.

Reply:
It is against the OASIS best practices: https://github.com/oasis-roles/meta_standards/blob/master/README.md#yaml-and-jinja2-syntax

Reply:
why they reported an error? The template properly avoids it.
https://github.com/linux-system-roles/template/blob/5bac4c7337af36b31c3191e4cd3aa4ddff2277bd/.yamllint.yml#L14

Reply:
ok - latest commit removes all leading `---` from yaml files

Reply:
the fact that you saw an error is still suspect

Reply:
> the fact that you saw an error is still suspect

No, the error makes sense to me.  Try it yourself.

Reply:
Ah, you saw a different error. Yes, `document-start: disable` disables the check for `---` at the beginning, not for blank lines. Blank lines are unrelated to the leading `---` marker. Removing the blank lines was right, they always seemed silly to me, I don't know why some YAML files have them.